scripts/releases/prebuild/cli.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ * @oncall react_native
+ */
+
+const {dependencies, platforms} = require('./configuration');
+const yargs = require('yargs');
+
+/*::
+import type {Dependency, Platform} from './types';
+import {skip} from "rxjs/operators";
+*/
+
+const arrayLike = (value /*: Array<any> */) /*: Array<any> */ =>
+  Array.isArray(value) ? value : [value];
+
+const cli = yargs
+  .usage(
+    'This script prepares iOS prebuilds for React Native. It downloads the dependencies, prepare them, builds them and creates the XCFrameworks.' +
+      'Calling the script with no options will build all the dependencies for all the slices and configurations.',
+  )
+  .option('setup', {
+    alias: 's',
+    type: 'boolean',
+    describe: 'Download and setup dependencies',
+  })
+  .option('swiftpackage', {
+    alias: 'w',
+    type: 'boolean',
+    describe: 'Creates the Package.swift file',
+  })
+  .option('build', {
+    alias: 'b',
+    type: 'boolean',
+    describe: 'Build dependencies/platforms',
+  })
+  .option('compose', {
+    alias: 'c',
+    type: 'boolean',
+    describe: 'Compose xcframework from built dependencies',
+  })
+  .option('platforms', {
+    alias: 'p',
+    type: 'array',
+    default: platforms,
+    describe: 'Specify one or more platforms to build for',
+  })
+  .option('dependencies', {
+    alias: 'd',
+    type: 'array',
+    default: dependencies.filter(d => !d.disabled).map(d => d.name),
+    describe: 'Specify one or more dependencies',
+  })
```

Comment:
we should add a 'configuration' parameter to choose whether to build for Debug or Release

Reply:
See above